src/mongo/db/curop.cpp

```diff
@@ -446,6 +446,8 @@ uint64_t CurOp::MaxTimeTracker::getRemainingMicros() const {
     return _targetEpochMicros - now;
 }
 
+PseudoRandom CurOp::_prng(std::unique_ptr<SecureRandom>(SecureRandom::create())->nextInt64());
```

Comment:
Yes, I know time(0) is a good old way. But for me using SecureRandom looks like a good new way. I found at least 4 places in upstream code where it is used this way. My guess is time(0) is monotonically increasing function and this is why it is worse than using SecureRandom for seed.
So I prefer to leave this part unchanged.